rapids-cmake/cpm/versions.json

```
@@ -16,7 +16,7 @@
       "version": "3.0.0",
```

This is a commit after the 3.0.1 tag so we should update the version. I’ll apply this fix.
```suggestion
      "version": "3.0.1",
```